src/app/util/ember-compatibility-functions.cpp

```diff
@@ -260,6 +338,22 @@ CHIP_ERROR ReadSingleClusterData(FabricIndex aAccessingFabricIndex, const Concre
                   "Reading attribute: Cluster=" ChipLogFormatMEI " Endpoint=%" PRIx16 " AttributeId=" ChipLogFormatMEI,
                   ChipLogValueMEI(aPath.mClusterId), aPath.mEndpointId, ChipLogValueMEI(aPath.mAttributeId));
 
+    if (aPath.mAttributeId == Clusters::Globals::Attributes::AttributeList::Id)
+    {
+        // This is not in our attribute metadata, so we just check for this
+        // endpoint+path existing.
```

Comment:
And so this is the third occurrence of ZAP support beeing not great, correct ? Since it is not generated into `endpoint_config.h`, we need custom code to handle it ?

Reply:
We'd need custom code not matter what, but yes, the fact that the check here is special and has to come before the attribute metadata check is because ZAP does not generate it into endpoint_config.

That said, adding this to endpoint_config will be somewhat bloaty.  :(